Examples/BinaryBH/BinaryBHLevel.cpp

```diff
@@ -83,18 +83,26 @@ void BinaryBHLevel::computeTaggingCriterion(FArrayBox &tagging_criterion,
 
 void BinaryBHLevel::specificPostTimeStep()
 {
-    if ((m_p.activate_extraction == 1) &&
-        (m_level == m_p.extraction_params.extraction_level))
+    CH_TIME("BinaryBHLevel::specificPostTimeStep");
+    if (m_p.activate_extraction == 1)
     {
         // Populate the Weyl Scalar values on the grid
         fillAllGhosts();
         BoxLoops::loop(Weyl4(m_p.extraction_params.extraction_center, m_dx),
                        m_state_new, m_state_new, EXCLUDE_GHOST_CELLS);
 
-        // Now refresh the interpolator and do the interpolation
-        m_gr_amr.m_interpolator->refresh();
-        WeylExtraction my_extraction(m_p.extraction_params, m_dt, m_time);
-        my_extraction.execute_query(m_gr_amr.m_interpolator);
+        // Do the extraction on the min extraction level
+        auto min_extraction_level_it =
```

Comment:
Minor cosmetic comment: could we move this logic to find the minimum value into the simulation parameters class and define m_p.min_extraction_level in there?
Since BinaryBHLevel is the first file a new user should look at I think it's worth trying to keep it as tidy as possible.